community/php7/php7-fpm.initd

```diff
@@ -52,21 +50,21 @@ start_pre() {
 	# directory exists and has correct privileges.
 	local listen="$(conf_get listen)"
 	if [ "${listen:0:1}" = "/" ]; then
-		checkpath -d -o $user:$group "$(dirname "$listen")"
+		checkpath -d -o $user:$group ${listen%/*}
```

Comment:
Why have you removed quotes around the `listen` variable? It’s a good practice to always quote variables that contain externally provided file path.